src/interaction-handlers/ticket/categorize.ts

```diff
@@ -54,7 +54,7 @@ export class ButtonHandler extends InteractionHandler {
 			await setTicketOpen(channel, true);
 			return interaction.update({
 				content: `${ticketTypeDesc}
-Go ahead and describe your problem so we can help you.
+Please describe your problem so we can help you.
```

Comment:
This same original phrasing is used in categorizeCrash. Should that be changed as well?

Reply:
yeah that'd work as well